SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -259,20 +260,6 @@ void GenerateSubobjectSchema(UClass* Class, TSharedPtr<FUnrealType> TypeInfo, FS
 			}
 		}
 
-		// If this class is an Actor, it MUST have bTearOff at field ID 3.
-		if (Group == REP_MultiClient && Class->IsChildOf<AActor>())
```

Comment:
How come we're removing this?

Reply:
I feel like there was a reason but I'm forgetting why. I thought I moved this somewhere.